### DISCUSSION TITLE
what it is HEADLESS login, HEADLESS wordpress?

### DISCUSSION THREAD

**Discussion by hhniao** [AxeWP/wp-graphql-headless-login#126] [Q&A] [answered]
as the title.

**Comment by justlevine** [comment, marked as answer]
Hey @hhniao 

Headless _WordPress_ is when you ditch the traditional WordPress Frontend entirely, and instead use APIs to populate your frontend of choice with WordPress data.

Headless Login for WPGraphQL is a plugin extension that provides multiple ways to authenticate with WordPress on a headless site, using the GraphQL API. You can learn more in the documentation included in this readme.
